apps/skateboard/config/dev.exs

```diff
@@ -35,8 +35,8 @@ config :phoenix, :stacktrace_depth, 20
 # Configure your database
 config :skateboard, Skateboard.Repo,
   adapter: Ecto.Adapters.Postgres,
-  username: "postgres",
-  password: "postgres",
+  username: "shamim",
```

Comment:
probably won't work for other people - can you change your db owner to "postgres"